File: BASIC/5-countDigit.js

// function countNum(num)
// {
//     var cnt =0;
//     while(num != 0)
//     {
//         num = Math.floor(num / 10);
//         cnt++;
//     }
//     return cnt;
// }
function countNum2(num)
{
   let s =  num.toString();
   return s.length;

}

var num = 55;
console.log(countNum2(num));

File: BASIC/5-countDigit.cpp
#include<iostream>
#include<bits/stdc++.h>
using namespace std;
int count(int n)
{
    int cnt = 0;
    while(n != 0)
    {
       n = n / 10; 
       cnt++;
    }
    return cnt;
}
int count2(int n)
{
    string s = to_string(n);
    return s.length();
}
int count3(int n)
{
    int digit = floor(log10(n) + 1);
    return digit;
}
int main()
{
    int n;
    cout << "Enter the number"<< endl;
    cin >> n;
    cout << "Count is =" << count3(n);
}